src/runtime_src/core/pcie/driver/linux/xocl/xocl_fdt.h

```diff
@@ -113,7 +113,7 @@
 #define NODE_PCIE "pcie"
 #define NODE_BARS "bars"
 #define NODE_XGQ_SQ_BASE "ep_xgq_mgmt_to_rpu_sq_pi_00"
-#define NODE_XGQ_RING_BASE "ep_xgq_payload_mgmt_00"
+#define NODE_XGQ_PAYLOAD_BASE "ep_xgq_payload_mgmt_00"
```

Comment:
Is NODE_XGQ**_VMR**_PAYLOAD_BASE better?

Reply:
yes, fixed.

Reply:
yes, fixed.